database/src/main/kotlin/no/nav/su/se/bakover/database/DatabaseEx.kt

```diff
@@ -13,9 +13,23 @@ private fun sjekkUgyldigParameternavn(params: Map<String, Any?>) {
     require(params.keys.none { it.contains(Regex("[æÆøØåÅ]")) }) { "Parameter-mapping forstår ikke særnorske tegn" }
 }
 
+private fun sjekkAtOppdaterInneholderWhere(sql: String) {
+    require(sql.lowercase().contains("where")) { "Ikke lov med update uten where" }
+    require(!sql.lowercase().contains("insert")) { "Ikke lov med insert i en update" }
+}
+
```

Comment:
Denne gjør litt mer enn den sier (siden den også sjekker at strengen ikke inneholder `insert`). Kanskje navnet heller kunne ha vært `sjekkAtSpørringErGyldigUpdate`? Kunne f.eks. også da sjekket at den faktisk inneholder `update`.
Kunne også gjort tilsvarende for `insert`; lag en metode `sjekkAtSpørringErGyldigInsert` som f.eks. sjekker at den inneholder `insert into` ellerno.

Reply:
Vi endte med å fjerne insert sjekken. 
Vi tenkte at poenget med oppgaven var vel å hindre at vi oppdaterte alle radene i en tabell ved en feil.